### ISSUE TITLE
Not receiving anything

### ISSUE THREAD

**Issue by zamzon** [jalla2000/kamstrup-mqtt#3]
Hi

I copy'ed the sketch.ino file into Arduino IDE and complied the file on my Wemos D1 mini
and connected the D7 and GND as shown in the picture.
But when i look in the Topic house/electricity/status i only get this counting up. 

Looping. Loops=81
Received bytes: 0

Do i need to set something else up in the sketch before compling the file?

**Comment by zamzon** [comment]
I also did try subscribing to house/electricity/# and nothing is received

**Comment by SalkinDK** [comment]
@zamzon excatly the same problem here. have tried all sort of things, but nothing help.
did you found a solution?
@jalla2000 any tips to what we can do?

**Comment by zamzon** [comment]
I haven’t found a solution yet 😩

**Comment by SalkinDK** [comment]
@zamzon me neither.

But i can see that @jalla2000 is based in Norway, so maybe that could be the problem?
i am based in Denmark, dont know about you?

**Comment by zamzon** [comment]
That could be the problem ☹️,
I’m based in DK as well

Øv bøv..

**Comment by MTrab** [comment]
It's encrypted here in Denmark - so this (which reads in cleartext) will not work in Denmark.

**Comment by zamzon** [comment]
Thank.
Does your ir solution work on this meter, with an ESP8266, maybe with MQTT

**Comment by MTrab** [comment]
No, not on ESP boards - it only works on Arduino.
Other than that, different providers seem to begin disabling IR in the meters.
Aura for one have disabled IR on their Omnipower meters - huray for GDPR :(

**Comment by zamzon** [comment]
🤔 so there is no solution you know that works perfectly?

**Comment by MTrab** [comment]
Not DIY - currently I'm testing Smart-me, but I experience really crappy range on the wifi

**Comment by SalkinDK** [comment]
> It's encrypted here in Denmark - so this (which reads in cleartext) will not work in Denmark.

Yes, i talked to Radius the other day, and they told me that they are using the encrypted (open) kamstrup protocol, and that you need a decryption key.

But he also told me that they are going to implement the DLMS protocol during this fall.
So there should be a change to have a proper esp solution be the end of the year.



> Not DIY - currently I'm testing Smart-me, but I experience really crappy range on the wifi

Yes, and not really a solution at least for me since all the data is pushed to the cloud. I would like a solution where the data never leaves my internal network.

**Comment by MTrab** [comment]
You can switch smart-me to modbus and close the firewall for the module, hence no data leaving the internal net ;)

**Comment by jesperldk** [comment]
Yes, I use modbus with the smart-me, works fine. 

Sendt fra min iPhone

**Comment by SalkinDK** [comment]
> You can switch smart-me to modbus and close the firewall for the module, hence no data leaving the internal net ;)

Hmm, ok any of you who know how that will integrate with homeassistant?
